cli/dcoscli/test/package.py

```diff
@@ -48,30 +52,25 @@ def package(package_name, deploy=False, args=[]):
 
 def setup_universe_server():
     # add both Unvierse servers with static packages
-    add_app('tests/data/universe-v3-stub.json', True)
-    add_app('tests/data/helloworld-v3-stub.json', True)
+    add_app('tests/data/universe-v3-stub.json', False)
+    add_app('tests/data/helloworld-v3-stub.json', False)
```

Comment:
What does False do here instead of True?

Reply:
it proceeds without watching for deployments, this is not needed as there is a `watch_all_deployments()` below.